jp/components/carousel.md

```diff
@@ -304,7 +304,7 @@ Carousel [アニメーション](carousel.md#angular-carousel-のアニメーシ
 </div>
 ...
 ```
-カルーセルの [`onSlideChanged`]({environment:angularApiUrl}/classes/igxcarouselcomponent.html#onSlideChanged) およびリストの [itemClicked]({environment:angularApiUrl}/classes/igxlistcomponent.html#itemClicked) イベントを処理し、コンポーネントを同期する方法: 
+カルーセルの [`slideChanged`]({environment:angularApiUrl}/classes/igxcarouselcomponent.html#slideChanged) およびリストの [itemClicked]({environment:angularApiUrl}/classes/igxlistcomponent.html#itemClicked) イベントを処理し、コンポーネントを同期する方法: 
 
```

Comment:
Could you please add this note ">[!NOTE]
>As of v15.1.0 `onSlideChanged` was renamed to `slideChanged`. Using `ng update` will automatically migrate your code prior to use the new event name.'"
to the jp and kr topics as well to have consistent content

Reply:
@randriova Has asked us to not include the EN content in the other language docs, but to just tag with `pending-localization`.